### PR TITLE
Added basic layout for roadmap

### PR DESCRIPTION
**Pull request by vinckr**
Basic layout for quick overview ORY roadmap.

### CODE REVIEW

File: docs/ecosystem/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Roadmap
+---
+
+## ORY Roadmap
+
+If you would like a quick overview of whats coming next to ORY, you can do so at our github project page.  
+You can see what we are working on right now, what issues are prioritized and what progress has been made in the past.  
+[Follow this link for the complete Project Roadmap](https://github.com/orgs/ory/projects/8).
+
+## Complete Roadmap Sections:
```
Comment (aeneasr) [inline docs/ecosystem/roadmap.md]:
```suggestion
## ORY Project Management Board
```

File: docs/ecosystem/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+id: roadmap
+title: Roadmap
+---
+
+## ORY Roadmap
+
+If you would like a quick overview of whats coming next to ORY, you can do so at our github project page.
```
Comment (aeneasr) [inline docs/ecosystem/roadmap.md]:
```suggestion
If you would like a quick overview of whats coming next to ORY, you can do so at our [GitHub Projects Board](https://github.com/orgs/ory/projects/8).  
```

File: docs/ecosystem/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+id: roadmap
+title: Roadmap
+---
+
+## ORY Roadmap
+
+If you would like a quick overview of whats coming next to ORY, you can do so at our github project page.  
+You can see what we are working on right now, what issues are prioritized and what progress has been made in the past.  
+[Follow this link for the complete Project Roadmap](https://github.com/orgs/ory/projects/8).
```
Comment (aeneasr) [inline docs/ecosystem/roadmap.md]: I think we can remove this as it's linked above.

File: docs/ecosystem/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
+[Done](https://github.com/orgs/ory/projects/8#column-6092772)  
+
+## Individual Project Sections
+
+[ORY Kratos Milestones](https://github.com/ory/kratos/milestones)  
+[ORY Kratos Pull Requests](https://github.com/ory/kratos/pulls)
```
Comment (aeneasr) [inline docs/ecosystem/roadmap.md]: You can combine PR and issues using filters like so: https://github.com/ory/kratos/issues?q=is%3Aopen

You can also sort by recent and updated recently: is:open sort:updated-desc sort:reactions-+1-desc

File: docs/ecosystem/roadmap.md (hunk 1 above)
Comment (vinckr) [reply docs/ecosystem/roadmap.md]: Did a Pr for that.

File: docs/ecosystem/roadmap.md (hunk 3 above)
Comment (vinckr) [reply docs/ecosystem/roadmap.md]: done

File: docs/ecosystem/roadmap.md (hunk 4 above)
Comment (vinckr) [reply docs/ecosystem/roadmap.md]: Ok, i sorted by "is open" now, it seems to be the most straightforward option.

File: docs/ecosystem/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
+[Done](https://github.com/orgs/ory/projects/8#column-6092772)  
+
+## Individual Project Sections
+
+[ORY Kratos Milestones](https://github.com/ory/kratos/milestones)  
+[ORY Kratos Pull Requests](https://github.com/ory/kratos/pulls?q=is%3Aopen)  
+[ORY Kratos Issues](https://github.com/ory/kratos/issues?q=is%3Aopen)
```
Comment (aeneasr) [inline docs/ecosystem/roadmap.md]:
```suggestion
```

File: docs/ecosystem/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
+[Done](https://github.com/orgs/ory/projects/8#column-6092772)  
+
+## Individual Project Sections
+
+[ORY Kratos Milestones](https://github.com/ory/kratos/milestones)  
+[ORY Kratos Pull Requests](https://github.com/ory/kratos/pulls?q=is%3Aopen)
```
Comment (aeneasr) [inline docs/ecosystem/roadmap.md]:
```suggestion
[ORY Kratos Open Issues and Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
```

**Comment by aeneasr** [comment]
> Please add this page to the navigation as discussed on slack

**Comment by aeneasr** [comment]
> Please add this page to the navigation as discussed on slack

Oh, I missed that you did this in another PR. That's ok, but you could do these changes in one PR as they belong to the same feature.

**Comment by aeneasr** [comment]
@vinckr are you still up for the changes here?

**Comment by vinckr** [comment]
> Oh, I missed that you did this in another PR. That's ok, but you could do these changes in one PR as they belong to the same feature.

Alright, ill keep it in mind!

Missed the change request, but ill get notifications from now on.

**Pull request merged**